src/Symfony/Component/RateLimiter/Rate.php

```diff
@@ -22,7 +22,14 @@
  */
 final class Rate
 {
+    /**
+     * @var \DateInterval
```

Comment:
When a property matches a constructor argument, we prefer removing these annotations.
The IDE understands the type correctly, and there is less room for typos. The less lines in a code base, the less PRs on it, the less maintenance for us.

Reply:
I'll remove some of these then. 

Thank you